dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/fileresource/DefaultFileResourceService.java

```diff
@@ -70,12 +66,6 @@
 
     public static final Predicate<FileResource> IS_ORPHAN_PREDICATE = (fr -> !fr.isAssigned());
 
-    private static final Set<String> CONTENT_TYPE_BLACKLIST = Sets.newHashSet( "text/html",
```

Comment:
Should we also move away from naming "Blacklist", and using "Blocklist" instead? 😄 
Read a lot of recent changes proposed in several companies/projects that want to move away from using "Black" as bad and "White" as good.

Reply:
Sure, changed to Blocklist.